.gitignore

```diff
@@ -74,3 +74,6 @@ custom_lint.sh
 
 # vscode
 .vscode
+
+# pyenv
+.python-version
```

Comment:
```suggestion
.vscode

# pyenv
.python-version




```
```suggestion
.vscode

# pyenv
.python-version
```
```suggestion
.vscode

# pyenv
.python-version
```
```suggestion
.vscode

# pyenv
.python-version
```

Reply:
Wow.  Ok, this computer sucks.  I would like a newline added to get rid of the red symbol.